admin/data/sql/seeds/20190129100803.custom.assign-familiarisation-form-to-check-windows.js

```diff
@@ -0,0 +1,39 @@
+module.exports.generateSql = () => {
+  if (process.env.NODE_ENV !== 'production') {
+    return `
+    DECLARE @CheckWindowCursor CURSOR;
+    DECLARE @checkWindowId INT;
+    BEGIN
+       SET @CheckWindowCursor = CURSOR FOR
+        -- FETCH ALL CURRENT OR FUTURE CHECK WINDOWS
+        SELECT cw.id from [mtc_admin].checkWindow cw
+        WHERE GETUTCDATE() <= checkEndDate
+        EXCEPT
+        -- EXCEPT THE ONES WHICH ALREADY HAVE MTC0103 FORM ASSIGNED
+        SELECT cw.id from [mtc_admin].checkWindow cw
+        LEFT JOIN [mtc_admin].checkFormWindow cfw
```

Comment:
Minor : I suspect that these LEFT JOINS can be normal JOINS

Reply:
@jon-shipley yes certainly. Updated